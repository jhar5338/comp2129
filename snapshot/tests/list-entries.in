list entries
set a
list entries
set b 1 1 2 3
set c 2 3
set 1 1
set 2 2
set 3 3
del 3
list entries
bye
