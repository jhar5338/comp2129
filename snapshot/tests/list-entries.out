> no entries

> ok

> a []

> ok

> ok

> ok

> ok

> ok

> ok

> 2 [2]
1 [1]
c [2 3]
b [1 1 2 3]
a []

> bye
